test/clustertest/tests/MultipleLeaderSyncTest.cpp

```diff
@@ -0,0 +1,94 @@
+#include "../BedrockClusterTester.h"
+
+struct MultipleLeaderSyncTest : tpunit::TestFixture {
+    MultipleLeaderSyncTest()
+        : tpunit::TestFixture("MultipleLeaderSyncTest",
+                              BEFORE_CLASS(MultipleLeaderSyncTest::setup),
+                              AFTER_CLASS(MultipleLeaderSyncTest::teardown),
+                              TEST(MultipleLeaderSyncTest::test)
+                             ) { }
+
+    BedrockClusterTester* tester;
+
+    // Create a bunch of trivial write commands.
+    void runTrivialWrites(int writeCount, int nodeID = 0) {
```

Comment:
could just pass the actual node into this instead of a nodeID since you made variables out of all of the nodes, makes the code a little easier to read.